mne/preprocessing/tests/test_otp.py

```diff
@@ -0,0 +1,104 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD 3 clause
+
+import os.path as op
+
+import pytest
+
+import numpy as np
+from numpy.fft import rfft, rfftfreq
```

Comment:
why not taking it from scipy ?

Reply:
SciPy's RFFT returns data in a different, more annoying form